src/App.tsx

```diff
@@ -65,6 +66,10 @@ function App() {
                     <Route index element={<StartPage />} />
                     <Route path="/items/:id" element={<DetailPage />} />
                     <Route path="/category/:name" element={<CategoryPage />} />
+                    <Route
```

Comment:
Why is the category 'Free' its own component but all other categories falls under 'categoryPage' component?

Reply:
because free category is not in the database its hard coded

Reply:
Aah ok! I also noticed that free isnt really a category. More of a way to sort within a category and thats nice!  :)

Reply:
Yeah, we thought it would  be the easiest way to be done :)